extension/src/completion/autocompletionProvider.ts

```diff
@@ -71,6 +86,7 @@ export function readAllBultinFunctions() {
             allCmdsAndSysvars.push(item)
     });
 
+    loadWebHelpAbstraction("../../extension/data/webHelpAbstraction.json");
```

Comment:
Our extension can be used in Mac, I see some vla- prefix functions and objects, which are not supported in AutoCAD MAC.

Maybe this is not critical, we can left it there.

Reply:
I don't think this particular point is an issue. If I am working on a Mac, I've borrowed some code online and I trying to figure out why its breaking at a vla command, I would want the help to open that documentation so I could see it plainly stated it isn't supported in official documentation. Make sense?

Reply:
this is not a big issue, we can tell testers to accept this.